config/environments/development.rb

```diff
@@ -56,10 +56,8 @@
   # Highlight code that triggered database queries in logs.
   config.active_record.verbose_query_logs = true unless Settings.status == "database_offline"
 
-  # Debug mode disables concatenation and preprocessing of assets.
-  # This option may cause significant delays in view rendering with a large
-  # number of complex assets.
-  config.assets.debug = true
```

Comment:
This change breaks my local development environment. Without the setting, I get html generated that refers to incorrect stylesheet paths. But if I add this setting back in, everything works.

Is there a reason for it being removed? If not, I would keep it for now.

I don't know why removing the setting breaks my environment, but until I figure it out I'm concerned that it might break things for other developers too.

Reply:
It was just because it was the new default from upstream. It didn't break it for me though I agree that it may make debugging a lot harder so I was surprised upstream had removed it. I guess it's related to the change in preferred methods for asset handling so I'll revert that for us.